python/dgl/nn/pytorch/conv/cugraph_relgraphconv.py

```diff
@@ -214,87 +87,68 @@ def __init__(
         regularizer=None,
         num_bases=None,
         bias=True,
-        activation=None,
         self_loop=True,
         dropout=0.0,
-        layer_norm=False,
-        max_in_degree=None,
+        apply_norm=False,
     ):
         if has_pylibcugraphops is False:
             raise ModuleNotFoundError(
-                "dgl.nn.CuGraphRelGraphConv requires pylibcugraphops "
+                "dgl.nn.CuGraphRelGraphConv requires pylibcugraphops >= 23.02 "
```

Comment:
Is there a way to check the version number of the package?

Reply:
https://github.com/rapidsai/dgl/blob/785e294b6e5a596df9c669eeb6ca56672a23d002/python/dgl/nn/pytorch/conv/cugraph_relgraphconv.py#L11-L13

Any version <23.02 does not have `pylibcugraphops.torch.autograd` API, and thus will throw an exception here.